.circleci/config.yml

```diff
@@ -47,12 +51,15 @@ jobs:
             - run:
                   name: Publish @pxblue/icons-svg
                   command: cd svg && yarn publish:package -b $CIRCLE_BRANCH
+            - run:
+                  name: Publish @pxblue/react-native-vector-icons
+                  command: cd rn-vector && yarn publish:package -b $CIRCLE_BRANCH
 workflows:
     version: 2
     icons:
         jobs:
             - build
-            - publish:
+            - publish_:
```

Comment:
This seems off, is the underscore intensional?

Reply:
good catch, not intentional